redfish-core/lib/systems.hpp

```diff
@@ -1583,9 +1583,10 @@ inline void
 
     std::string powerMode;
     const std::vector<std::string>* allowedModes = nullptr;
+    bool safeMode;
```

Comment:
Should this be defined as optional (to check the existence of the field)?
```
const bool* safeMode = nullptr;
```
OR
```
std::optional<bool> safeMode = nullptr;
```

Reply:
I'm not sure.. If it doesn't exist it hits the success and throws an error. I think that is what we had for 1060 
https://github.com/ibm-openbmc/bmcweb/blob/1060/redfish-core/lib/systems.hpp#L1547


```
    if (!success)
    {
        messages::internalError(asyncResp->res);
        return;
    }
```

Reply:
Oh well, I am good with the optional and just leave off if it doesn't exist